RAKE/RAKE.py

```diff
@@ -27,19 +27,34 @@ def FoxStopList():
     return FoxStopList.words()
 
 
-def load_stop_words(stop_word_file):
+def MySQLStopList():
+    from .stoplists import MySQLStopList
+    return MySQLStopList.words()
+
+
+def NLTKStopList():
+    from .stoplists import NLTKStopList
+    return NLTKStopList.words()
+
+
+def load_stop_words(stop_word_file, divide, delimiter):
     """
     Utility function to load stop words from a file and return as a list of words
     @param stop_word_file Path and file name of a file containing stop words.
     @return list A list of stop words.
     """
     stop_words = []
-    for line in open(stop_word_file):
-        for word in line.split():  # in case more than one per line
-            stop_words.append(word)
+    if divide:
```

Comment:
So basically: 

Change functions flag to regex = "[\W\n]+" (that isn't a keyword right? It one according to google), remove the additional .run() functionality that that replaces, and reflect this in the documentation

then do:
```
def load_stop_words(stop_word_file, regex):
    # This would be default value of delimiter, matches non-whitespace non-newlines
    delimiter = "[\W\n]+"  
    with open(stop_word_file) as f:
    words = re.split(delimiter, f.lines)
return [word for word in words if word not in ('', ' ')]  # filters empty string matches 
```

My only concerns are 
1) the .lines seems like it break the functionality of being able to use regex to not split lines?
2) Can you please change f, word and words to less opaque variables in that?